pulsar-checksum/src/main/java/com/yahoo/pulsar/checksum/utils/Crc32cChecksum.java

```diff
@@ -17,21 +17,23 @@
 
 import static com.scurrilous.circe.params.CrcParameters.CRC32C;
 
-import java.nio.ByteBuffer;
-
 import com.scurrilous.circe.IncrementalIntHash;
 import com.scurrilous.circe.crc.Sse42Crc32C;
 import com.scurrilous.circe.crc.StandardCrcProvider;
 
 import io.netty.buffer.ByteBuf;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class Crc32cChecksum {
 
+    private static final Logger log = LoggerFactory.getLogger(Crc32cChecksum.class);
     private final static IncrementalIntHash CRC32C_HASH;
 
     static {
         if (Sse42Crc32C.isSupported()) {
             CRC32C_HASH = new Crc32cSse42Provider().getIncrementalInt(CRC32C);
+            log.info("SSE4.2 CRC32C provider initialized");
```

Comment:
actually, as it logs only once when broker starts. So, can we keep it "INFO" initially to get confirmation about broker loaded library successfully and it is not computing checksum using slower-software-algo .